packages/vite/src/node/plugins/worker.ts

```diff
@@ -75,7 +87,7 @@ async function serialBundleWorkerEntry(
   const bundle = await rollup({
     ...rollupOptions,
     input: cleanUrl(id),
-    plugins,
+    plugins: plugins.concat(workerPostPlugin),
```

Comment:
I wonder if we can avoid creating a new Array here.
For example, adding `workerPostPlugin` here.
https://github.com/vitejs/vite/blob/050c0f9661231737797290c1c1faa761d98e00f3/packages/vite/src/node/build.ts#L448
```js
 ...(config.isWorker ? [workerPostPlugin] : []),
```

Reply:
Yeah, that would be better